integration-test/src/test/java/org/terracotta/angela/UnuspportedVersionIT.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Terracotta, Inc.
+ * Copyright IBM Corp. 2024, 2025
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.angela;
+
+import org.hamcrest.Matcher;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.terracotta.angela.client.AngelaOrchestrator;
+import org.terracotta.angela.client.Tsa;
+import org.terracotta.angela.client.support.junit.AngelaRule;
+import org.terracotta.angela.common.topology.Topology;
+import java.time.Duration;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.is;
+import static org.terracotta.angela.client.config.custom.CustomConfigurationContext.customConfigurationContext;
+import static org.terracotta.angela.common.TerracottaConfigTool.configTool;
+import org.terracotta.angela.common.distribution.Distribution;
+import org.terracotta.angela.common.distribution.Distribution107InlineController;
+import org.terracotta.angela.common.net.DefaultPortAllocator;
+import org.terracotta.angela.common.net.PortAllocator;
+import static org.terracotta.angela.common.dynamic_cluster.Stripe.stripe;
+import static org.terracotta.angela.common.provider.DynamicConfigManager.dynamicCluster;
+import static org.terracotta.angela.common.tcconfig.TerracottaServer.server;
+import org.terracotta.angela.common.topology.LicenseType;
+import org.terracotta.angela.common.topology.PackageType;
+import org.terracotta.angela.common.topology.Version;
+import org.terracotta.angela.common.util.IpUtils;
+import org.terracotta.angela.util.Versions;
+import net.schmizz.sshj.common.IOUtils;
+
+public class UnuspportedVersionIT {
```

Comment:
This test name doesn't seem to match what it's doing.

Reply:
Yes it is badly named. It was named like that because I wanted to set the version to 6.0.0, but then it tried to download ehcache kit 6.0.0 which does not exist...  That is a test that it impossible to do unless we have a new major release in platform or ehcache.